.cargo/config.toml

```diff
@@ -4,6 +4,9 @@ ci-clippy = "clippy --all-targets --all-features -- -D warnings"
 ci-license = "license-template --template .license_template"
 ci-udeps = "udeps --all-targets --backend=depinfo"
 ci-udeps-external = "udeps --all-targets --manifest-path external-crates/move/Cargo.toml --backend=depinfo"
+ci-check-custom-indexer-example = "check --manifest-path examples/custom-indexer/rust/Cargo.toml"
+ci-check-tic-tac-toe-example = "check --manifest-path examples/tic-tac-toe/cli/Cargo.toml"
```

Comment:
Doesn't seem very pragmatic to have to add one command per example :(

Reply:
That's true. I'll find a better way.

Reply:
I would rather not have these commands here and iterate over the result of a `find` in the workflow

Reply:
Okay, I made that change